libraries/redcore/database/driver.php

```diff
@@ -248,5 +247,203 @@ public static function deleteInstances()
 		{
 			self::$instances = null;
 		}
+
+		/**
+		 * Splits a string of multiple queries into an array of individual queries.
+		 *
+		 * @param   string  $sql  Input SQL string with which to split into individual queries.
+		 *
+		 * @return  array  The queries from the input string separated into an array.
+		 *
+		 * @since   11.1
+		 */
+		public static function splitSql($sql)
```

Comment:
calling RDatabaseDriver::splitSQL in j25 and j34 end up in a different files... https://github.com/redCOMPONENT-COM/redCORE/blob/develop/libraries/redcore/database/driver.php#L12